src/services/user.js

```diff
@@ -14,6 +14,15 @@ const getUser = async (usernameOrId) => {
   return user;
 };
 
+const getFriends = async (id) => {
+  const users = await models.users.findAll({
+    where: {
+      id,
```

Comment:
Burada kullanıcının hash ve salt değeri de gelecektir. Bunların kullnıcı güvenliği için veri içinden kalırılması lazım